deblur/test/test_workflow.py

```diff
@@ -91,8 +90,11 @@ def test_sequence_generator_fastq(self):
         self.assertTrue(obs[0][1].startswith(first_few_nucs))
 
     def test_sequence_generator_invalid_format(self):
-        with self.assertRaises(skbio.io.FormatIdentificationWarning):
-            next(sequence_generator(join(self.test_data_dir, 'readme.txt')))
+        allres = []
+        with self.assertWarns(UserWarning):
```

Comment:
Agree with @mortonjt if you're testing that the warning is raised, you need to make sure that the warning that you are catching here is the one that you're raising